metrics/network/remote_network/remote-networking-cpu.sh

```diff
@@ -62,8 +62,6 @@ function remote_network_cpu_consumption {
 	test_name="Remote Network CPU Consumption"
 	units="%"
 	get_runtime
-	init_env
```

Comment:
/me mumbles that we don't normally wrap the subject line of commits into the body text... ;-)

Reply:
/me then mumbles at github gui for showing that as a split line, when in fact it is not (although it may be a little long).... :-/